updatecli/updatecli.d/jdk17.yml

```
@@ -0,0 +1,54 @@
+---
+title: "Bump JDK17 version"
+sources:
+  JDK17Version:
+    # Version is retrieved from the GitHub source mirror: if there is a release, then you're sure that the artifacts are available
+    kind: githubRelease
+    name: Get the latest Adoptium JDK17 version
+    spec:
+      owner: "adoptium"
+      repository: "temurin17-binaries"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionFilter:
+        kind: regex
+        # jdk-17.0.12+7(https://github.com/adoptium/temurin17-binaries/releases/tag/jdk-17.0.12%2B7) is OK
+        pattern: "^jdk-17+(\\d*)$"
```

not 100% sure I did the right thing here

---

```suggestion
        pattern: "^jdk-17\\+(\\d*)$"
```

The character `+` must be escaped in the regex to be treated as a litteral